ts-tests/build/IERC20Metadata.json

```diff
@@ -0,0 +1,244 @@
+{
```

Comment:
do we want to include the build folder?

Reply:
Prepare to delete the `modules/evm-bridge/src/erc20_demo_contract` file and reference it from the build directory, but there is a problem